clock_exercise/TickOfTopAlarmButton.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick 2.0
+
+Item {
+    id: root
+
+    property bool visualizationOfSelected
+    property bool visualizationOfHover
+
+    width: 34
+    height: 34
+
+    Image {
+        opacity: (root.visualizationOfSelected && !root.visualizationOfHover || !root.visualizationOfSelected && root.visualizationOfHover)? 1:0
```

Comment:
Attenzione! Questo non solo è un po' difficile da leggere, ma moltro probabilmente non fa quello che ti aspetti: a giudicare anche dall'effetto di hover quando ci vado sopra col mouse deduco che sia proprio così infatti.

Suppongo che il controllo fosse `(root.visualizationOfSelected && !root.visualizationOfHover) || (!root.visualizationOfSelected && root.visualizationOfHover)`, nota come ho racchiuso le due parti tra le parentesi.
Così come è nel codice vengono eseguiti gli operatori booleani in sequenza anzichè a gruppi di due.

Io semplificherei il tutto così, senza operatore ternario e con una condizione molto più leggibile:
`opacity: root.visualizationOfSelected || root.visualizationOfHover`

Reply:
![image](https://user-images.githubusercontent.com/70155823/96007346-b1e0e800-0e3e-11eb-84ed-1c421541e8c5.png)